docs/photutils/psf.rst

```diff
@@ -13,20 +13,33 @@ called "PSF photometry".
 
 Terminology
 -----------
-Different astronomy sub-fields use the terms Point Spread Function (PSF) and
-Point Response Function (PRF) somewhat differently, especially when colloquial
-usage is taken into account.  For this module we assume that the PRF is an image
-of a point source *after discretization* e.g., onto a rectilinear CCD grid. This
-is the definition used by `Spitzer
+Different astronomy sub-fields use the terms "PSF", "PRF", or related terms
+somewhat differently, especially when colloquial usage is taken into account.
+This package aims to be at the very least internally consistent, following the
+definitions described here.
+
+For this module we take Point Spread Function (PSF), or instrumental Point Spread
+Function (iPSF) to be the infinite resolution and infinite signal-to-noise flux
+distribution from a point source on the detector, after passing through optics,
+dust, atmosphere, etc. By contrast, the function describing the efficiency of
+the *pixels* of the detector is the Pixel Response Function (sometimes called
```

Comment:
Nitpicking: the words `describing the efficiency of the detector` makes it sound as if the PRF also folds in the flat-field, i.e. the intra- and inter-pixel sensitivity variations of the detector.  Is this the intended meaning?

Reply:
I agree that the current terminology is a bit vague.  It would be good to explicitly state that the "Pixel Response Function" means the responsivity variations across a single pixel.

Reply:
Good point - I was meaning just what @karllark says, but @barentsen is right it's unclear.  Will update in the next round.